src/cli/submit/data-set/DataSet.definition.ts

```diff
@@ -27,14 +27,26 @@ export const SubmitDataSetDefinition: ICommandDefinition = {
             description: "Submit a job from the data set \"ibmuser.cntl(iefbr14)\" and print only the job ID",
             options: "\"ibmuser.cntl(iefbr14)\" --rff jobid --rft string"
         },
+        {
+            description: "Submit a job from the data set \"ibmuser.cntl(iefbr14)\" and wait for job complete.",
+            options: "\"ibmuser.cntl(iefbr14)\" --wait 5,12"
+        },
     ],
     positionals: [{
         name: "dataSet",
         description: "The data set containing JCL that you would like to submit",
         type: "string",
         required: true
     }],
-    options: [],
+    options: [
+        {
+            name: "wait",
+            aliases: ["w"],
+            description: "Specify job query interval and max times of querying job status. " +
+            "The default value is '5,12' which queries job status every 5 seconds for 12 times at most.",
```

Comment:
Should there be more information about the format of the `--wait` flag (e.g comma-separated numeric values)?

Reply:
Yes. I will add the format information in this description.